src/pages/Facility/settings/organizations/components/EditFacilityUserRoleSheet.tsx

```diff
@@ -155,18 +155,22 @@ export default function EditUserRoleSheet({
               </div>
             </div>
 
-            <div className="grid grid-cols-2 gap-4 pt-2 border-t">
-              <div>
-                <span className="text-sm text-gray-500">{t("username")}</span>
-                <p className="text-sm font-medium">{userRole.user.username}</p>
+            <div className="flex flex-col flex-wrap gap-2 pt-2 border-t">
```

Comment:
<img width="303" alt="image" src="https://github.com/user-attachments/assets/c88f5701-4e97-4fa6-9ca7-df6b239dec53" />

Let's add truncate and tooltip for usernames (refer to other examples in codebase).